modules/appeals_api/app/models/appeals_api/appellant.rb

```diff
@@ -40,6 +40,12 @@ def insurance_policy_number
       auth_headers['X-VA-Insurance-Policy-Number']
     end
 
+    def birth_date
+      return if birth_date_string.blank?
+
+      @birth_date ||= Date.parse(birth_date_string)
+    end
+
     def birth_month
```

Comment:
Will remove the birthday formatting methods altogether in a following PR since they will add form_data methods to HLR v2.